ballerina/README.md

```diff
@@ -68,6 +90,16 @@ Create a Snowflake client endpoint by giving authentication details in the Snowf
 snowflake:Client snowflakeClient = check new(accountIdentifier, user, password);
 ```
 
+If you are using key-pair authentication, you can create a Snowflake client endpoint as follows:
+```ballerina
+AuthConfig authConfig = {
+    user,
```

Comment:
Better to add a sample with mock values rather than providing variables which matches the field names
```ballerina
AuthConfig authConfig = {
    user: "ballerina"
    privateKeyPath: "path/to/privatekey.p12"
    privateKeyPassphrase: "ballerina"
};

snowflake:AdvancedClient snowflakeKeyBasedClient = check new (accountIdentifier, authConfig, options);
```